### PR TITLE
Signup page design

### PR DESCRIPTION
**Pull request by adejam**
# Project: INTUIT SIGNUP PAGE DESIGN
The main objective of this project is to build a clone of Intuit website's signup page clone using Html and CSS.

The project uses semantic html and  HTML5 form with the appropriate input types for email, password and telephone inputs. It also uses the w3c recommend input and label association for better accessibility. CSS margin property in conjunction with CSS float property is used for effective positioning of the elements.

This project demonstrates usage of html forms and CSS positioning techniques.

- [Assignment link](https://www.theodinproject.com/courses/html5-and-css3/lessons/html-forms)
- [Intuit signup page](https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Ftask%3DS)
- [Live link](https://adejam.github.io/intuit-signup-page-clone/index.html)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
-# intuit-login-page-clone
-This is a clone of intuit website's login page clone
+# intuit-signup-page-clone
+This is a clone of intuit website's signup page clone
+
+## BUILT WITH
+
+- HTML
+
+- CSS
```
Comment (bolah2009) [inline README.md]: [Optional]

We can still list the features of HTML/CSS we used, for example, CSS flexbox or HTML forms.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+
+## Live Demo
+- [Live Demo Link](https://adejam.github.io/intuit-signup-page-clone/index.html)
+
+## Images
+![Youtube video page clone](/images/signup-page.png)
+## Screenshot: 
+![Youtube video page clone](/images/signup-page.png)
```
Comment (bolah2009) [inline README.md]: [Optional]

Is this not the same, is there a reason why you are repeating the same picture?

I would recommend using only one. Also, the alt should be about the current project not **Youtube video page clone**.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,37 @@
...
+
+## Contributing
+Contributions, issues and feature requests are welcome!
+
+
+## Show Your Support
+Give a star if you like this project!
+
+
+## Acknowledgments
```
Comment (bolah2009) [inline README.md:42]: [optional]

I think you can also reference the webpage you cloned and Odin project where the assignment is from. :thinking:

File: README.md [hunk 4]
```diff
@@ -1,2 +1,37 @@
-# intuit-login-page-clone
-This is a clone of intuit website's login page clone
+# intuit-signup-page-clone
+This is a clone of intuit website's signup page clone
```
Comment (bolah2009) [inline README.md]: For users that don't know this website or want to learn more about the project, why not add the assignment link, the live link and the link of the webpage that is been a clone.

You could also add a brief description of **what, how and why** you worked on this project.

File: index.html [hunk 5]
```diff
@@ -1,15 +1,141 @@
...
-		<title></title>
-		<style >
-			
-		</style>
+		<title>Intuit. Signup</title>
 		<link rel="stylesheet" href="stylesheet.css" />
 	</head>
 	<body>
-		<h1>Intuit Login Page clone</h1>
+		<section>
```
Comment (bolah2009) [inline index.html]: You seems to be overusing the `section` tag, note that the `section` tag should **mostly** be used only for a sectional part of the page that has its own heading, that way your section element, will have it's own `heading` element.

See this errors/warning for more details https://validator.w3.org/nu/?doc=https%3A%2F%2Frawcdn.githack.com%2Fadejam%2Fintuit-signup-page-clone%2F3588299e896dbe4658a418577085bc0de193a739%2Findex.html

File: index.html [hunk 6]
```diff
@@ -1,15 +1,141 @@
...
+								<li class="list_item list_icon" id="cb_icon"></li>
+								<li class="list_item list_text" id="cb_text"></li>
+							</ul>
+						</div>
+					</header>
+				</div>
+				<div class="main">
+					<section >
+						<header class="form_header">
+							<span class="form_title">Create an Intuit account</span>
```
Comment (bolah2009) [inline index.html]: Don't you think this is a heading element rather than a `span`? :thinking:

File: index.html [hunk 7]
```diff
@@ -1,15 +1,141 @@
...
+							<span class="form_title_des">
+								<span class="div">One account for everything Intuit, including Mint.
+									<a href="#" class="link">Learn more</a>
+								</span>
+							</span>
+						</header>
+						<form>
```
Comment (bolah2009) [inline index.html:55]: #### 👍🏾

Nice job using `form` element, this makes the section represent a document section that contains interactive controls for submitting information to a web server. 👏🏾

File: index.html [hunk 8]
```diff
@@ -1,15 +1,141 @@
...
+							<fieldset>
+								<label class="label">Email address</label>
+								<div class="form_div">
+									<input type="email" name="email" class="form_input" />
+								</div>
```
Comment (bolah2009) [inline index.html]: :bulb: Consider associating the `<label>` with the `<input>` element, to do this you need to give the `<input>` an `id` attribute. The `<label>` then needs a `for` attribute whose value is the same as the input's id. 
- Example 1: 

``` html
  <label for="email">Email address</label> <br>
  <input type="email" name="email" id="email">
```
- Example 2: Alternatively, you can nest the `<input>` directly inside the `<label>` like this: 

``` html
  <label for="email">Email address <br>
       <input type="email" name="email" id="email">
  </label> 
```

> #### Associating a `<label>` with an `<input>` element offers some major advantages:
> 
> - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
> 
> - You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

You can learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)


Please do this for other input elements.

File: index.html [hunk 9]
```diff
@@ -1,15 +1,141 @@
...
+								<span class="input_des"></span>
+							</fieldset>
+							<fieldset>
+								<label class="label">Phone (recommended)</label>
+								<div class="form_div">
+									<input type="text" name="number" class="form_input" />
+								</div>
+								<span class="input_des">Standard call, messaging or data rates may apply.</span>
+							</fieldset>
```
Comment (bolah2009) [inline index.html]: See previous comment

File: index.html [hunk 10]
```diff
@@ -1,15 +1,141 @@
...
+							<fieldset>
+								<div class="form_div">
+									<label class="label">Password</label>
+								</div>
+								<div class="form_div">
+									<input type="password" name="password" class="form_input" />
+									<span class="password_lock"></span>
+								</div>
```
Comment (bolah2009) [inline index.html]: See previous comment

File: index.html [hunk 11]
```diff
@@ -1,15 +1,141 @@
...
+								<span class="input_des"></span>
+							</fieldset>
+							<div class="submit_div">
+								<button class="submit_button">
```
Comment (bolah2009) [inline index.html]: Note that the `type` attribute of a button is important, you can associate the button with the form by giving it a `type` attribute of `submit` or make it as it is by giving it an attribute of `button`.  Learn more here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button

File: index.html [hunk 12]
```diff
@@ -1,15 +1,141 @@
...
+								<label class="label">Email address</label>
+								<div class="form_div">
+									<input type="email" name="email" class="form_input" />
+								</div>
+								<span class="input_des"></span>
+							</fieldset>
+							<fieldset>
+								<label class="label">Phone (recommended)</label>
+								<div class="form_div">
+									<input type="text" name="number" class="form_input" />
```
Comment (bolah2009) [inline index.html]: The type attribute should be associated with `telephone`,  I would recommend going through the doc here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input to find an appropriate `type` attribute for a telephone number input.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
Awesome job with [the clone](https://rawcdn.githack.com/adejam/intuit-signup-page-clone/3588299e896dbe4658a418577085bc0de193a739/index.html) 👏🏾👏🏾👏🏾

Please check out the comments below

I would recommend adding more details of the project to both the PR description and README, I added suggestion of what could be added in the comments below.

Also, let's be more cautious of the commit messages, this is very important, the commit message needs to be descriptive of what is done it the respective commit. See [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) for more details.

File: README.md (hunk 1 above)
Comment (adejam) [reply README.md]: Thanks for the comment. I'll add that

File: README.md (hunk 2 above)
Comment (adejam) [reply README.md]: Thanks for the review. I'll work on it.

File: README.md (hunk 3 above)
Comment (adejam) [reply README.md:42]: Thanks for the comment. I'll add the reference to the webpage that was cloned and the Odin project where the assignment is from.

File: README.md (hunk 4 above)
Comment (adejam) [reply README.md]: Thanks for the comment. I'll add this.

File: index.html (hunk 5 above)
Comment (adejam) [reply index.html]: Thanks for the review. I'll remove all sections without its own headings.

File: index.html (hunk 8 above)
Comment (adejam) [reply index.html]: Thanks for the review. I'll work on this.

File: index.html (hunk 11 above)
Comment (adejam) [reply index.html]: Thanks for your comment. I'll check the link and add the appropriate type to the button.

File: index.html (hunk 12 above)
Comment (adejam) [reply index.html]: Thanks for your comment. I'll check the link and add the appropriate type to the input.

**Comment by adejam** [comment]
Thanks for your reviews and comments. The project has been improved on based on your reviews and comments.

**Pull request merged**